src/asciidoc/reference/packages/withkojiCustomVccSdk.adoc

```diff
@@ -6,46 +6,40 @@ The @withkoji/custom-vcc-sdk package enables you to
 implement custom Visual Customization Controls (VCCs) for your template.
 //end::description[]
 
```

Comment:
Happy to make this change but I want to explain my justification for not doing that currently. I see the docs section as an area to better understand what Koji can do and what to use where and is more useful to new-comers; I see the refs section as a place established devs would come to read up on functions they've forgot or check return types in packages they are already aware of, this kind of thing.

I think by adding too much info in the ref section preamble you're making it more like the first port of call for that package which seems to break that pattern. I imagined if we find that a package isn't self explanatory within the references, instead of adding more content there, it would suggest a tutorial of some sort is needed in the docs section.

Let me know what you think.

Reply:
Yes, I agree with everything you said, and definitely don't want to make the reference into a tutorial hybrid. However, we still need to provide enough context for someone to evaluate whether they're in the right place. I might have suggested the wrong direction with the "basic workflow." We probably just need a few sentences saying, at a high level, what this is and why you'd use it. Right now, it just says the minimum that was needed to list it on the index page (which doesn't do a very good job distinguishing it from the VCC package, for example). You can probably rephrase some of the text from the VCC overview and/or custom VCC tutorial pages. Speaking of which, we should add a link from the VCC overview to this page :)